src/includes/getting-started-primer/apple.macos.mdx

```diff
@@ -0,0 +1,27 @@
+<Note>
+
+Sentry's SDKs report an error automatically whenever a thrown error or exception goes uncaught in your application causing the application to crash. Sentry's iOS, tvOS, and watchOS SDK hook into all signal and exception handlers, except for MacOS.
+
+The SDK builds a crash report that persists to disk and tries to send the report right after the crash. Since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.
+
+</Note>
+
+**Features:**
+
+- Multiple types of errors are captured, including:
```

Comment:
The common entries could be moved [into an `include`](https://github.com/getsentry/sentry-docs/tree/master/src/includes) and then used anywhere it's needed.

Here's an example: https://github.com/getsentry/sentry-docs/tree/master/src/includes/configuration/diagnostic-logger

That can then be used in different platforms even:

https://github.com/getsentry/sentry-docs/blob/31b2ea2a13fd01b1929248199c82cf8f463d72e3/src/platforms/unity/common/configuration/diagnostic-logger.mdx#L7

https://github.com/getsentry/sentry-docs/blob/31b2ea2a13fd01b1929248199c82cf8f463d72e3/src/platforms/dotnet/common/configuration/diagnostic-logger.mdx#L7